scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
+
+def g_r(response, args):
+    if args[0] == 'time':
+        if len(args)!=4:
+            return ["ERROR: Please use the following syntax `time <channel> 2/1/17 5/1/18`"]
+        return time(args[2::], args[1])
+
+    if args[0] == 'top_users':
+        pass
+
+    if args[0] == 'emoji':
+        pass
+
+    if args[0] == 'channel':
+        pass
+
+    if args[0] == 'help':
+        return ["""*List of arguments for GenerateReport (g_r)*
+```
+time (g_r time random 12/01/15 5/01/17)
+    -- Returns time related data on the channel specified, if none given
+    then default is slack-wide.
+    -- Takes 3 arguments: <channel> <begin date> <end date>```
+        """]
+
+
+def time(range, channel='all'):
```

Comment:
does this create a time graph? If so could we go with `create_time_graph` or `generate_time_graph`, nouns should generally be reserved for objects, while funcitons should start with a verb when possible